tests/unit/mathematics/Random_unittest.cc

```diff
@@ -312,35 +320,37 @@ TEST(Random, random_float64_range)
 
 TEST(Random, random_float64_range2)
 {
-	CMath::init_random(12345678);
+	CRandom* prng = new CRandom(12345678);
 	float64_t min=1.0;
 	float64_t max=0.0;
 	for (uint32_t i=0; i<n_runs; i++)
 	{
-		float64_t r=CMath::random((float64_t) 0, (float64_t) 1.0);
+		float64_t r = prng->random((float64_t)0, (float64_t)1.0);
 		min=CMath::min(min, r);
 		max=CMath::max(max, r);
 	}
 	EXPECT_GE(max, 0.99999);
```

Comment:
I have to say, these are very *nice* test to ensure correctness of randomness :D